### ISSUE TITLE
Error: MiniProfiler storage failure: Invalid argument @ rb_sysopen

### ISSUE THREAD

**Issue by ghost** [MiniProfiler/rack-mini-profiler#145]
I added to my Gemfile this:

``` ruby
group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug'
  gem "better_errors"
  gem 'bullet'
  gem 'lol_dba'
  #gem 'faker'
  gem 'quiet_assets'
  gem 'rails-footnotes'
  gem 'brakeman', :require => false
  gem 'rails_best_practices', :require => false
   # Access an IRB console on exception pages or by using <%= console %> in views
  # gem 'web-console', '~> 2.0'
  gem 'web-console'
  gem 'rack-mini-profiler'
end
```

But I have this error:

**MiniProfiler storage failure: Invalid argument @ rb_sysopen - L:/web/tmp/miniprofiler/mp_views_::1**

I'm on Windows 8.1 with Rubyinstaller 2.1.6 installed (everything else works good).

Why this error?

**Comment by SamSaffron** [comment]
AHA, never tested this in windows, looks like a bug its using a filename
windows does not like, PR to fix welcome.

**Comment by CromonMS** [comment]
Can confirm same error in Win 7
MiniProfiler storage failure: Invalid argument @ rb_sysopen - */tmp/miniprofiler/mp_views_::1

**Comment by jonathanhefner** [comment]
I believe this has been fixed by #157.

**Comment by SamSaffron** [comment]
yeah, very likely the storage engine issue.
